### ISSUE TITLE
dapp node rpc not found

### ISSUE THREAD

**Issue by dondaner** [dappnode/DAppNodePackage-bitcoin#47]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**DAppNode version:**

- Package version: 
- OS: 
- Browser 

**Additional context**

**Comment by dondaner** [comment]
btc node rpc port not defined 
what do i do

**Comment by Sigri44** [comment]
Same

**Comment by BenoistP** [comment]
Same issue started ~6 weeks ago
Bitcoin node seems to run fine but looks like Dashboard can't connect to it

**Comment by oOthkOo** [comment]
Same
![Capture d’écran du 2024-11-08 19-21-46](https://github.com/user-attachments/assets/a7af407a-308a-4f86-82f0-fd7bd39dcfa4)

**Comment by gitlappi** [comment]
Same issue, hope this could be fixed ASAP so I can keep my wallet connected to my node.

**Comment by oOthkOo** [comment]
It seems this PR fix the issue 

https://github.com/dappnode/DAppNodePackage-bitcoin/pull/46

Can you merge it and re-publish this package @alexpeterson91 @vdo @eduadiez ?

**Comment by SeeBeyondTheEventHorizon** [comment]
I am experiencing the same issue:  Dashboard error:  "RPC port not defined".  Is there a fix to the package in the works?

**Comment by Sigri44** [comment]
> I am experiencing the same issue: Dashboard error: "RPC port not defined". Is there a fix to the package in the works?

Yes on https://github.com/dappnode/DAppNodePackage-bitcoin/pull/48

**Comment by SeeBeyondTheEventHorizon** [comment]
Are folks still experiencing this issue?  I.e., Dashboard error "RPC port not defined"?  I am still experiencing the error.  It looks like some folks were working on a solve at one time.

**Comment by Sigri44** [comment]
> Are folks still experiencing this issue? I.e., Dashboard error "RPC port not defined"? I am still experiencing the error. It looks like some folks were working on a solve at one time.

fixed by PR but not included.... So migrate to Umbrel or Avado (:

**Comment by BenoistP** [comment]
> > Are folks still experiencing this issue? I.e., Dashboard error "RPC port not defined"? I am still experiencing the error. It looks like some folks were working on a solve at one time.
> 
> fixed by PR but not included.... So migrate to Umbrel or Avado (:

Umbrel doesn't allow to install on an existing OS anymore
https://github.com/getumbrel/umbrel/issues/1826#issuecomment-2371576359
but fortunately there's a docker version
https://community.umbrel.com/t/new-umbrelos-in-a-docker-container/19322/13

**Comment by Sigri44** [comment]
> > > Are folks still experiencing this issue? I.e., Dashboard error "RPC port not defined"? I am still experiencing the error. It looks like some folks were working on a solve at one time.
> > 
> > 
> > fixed by PR but not included.... So migrate to Umbrel or Avado (:
> 
> Umbrel doesn't allow to install on an existing OS anymore [getumbrel/umbrel#1826 (comment)](https://github.com/getumbrel/umbrel/issues/1826#issuecomment-2371576359) but fortunately there's a docker version https://community.umbrel.com/t/new-umbrelos-in-a-docker-container/19322/13
> 
> it runs fine on dappnode, just had to remap http 80 to 8080 for accessing Umbrel GUI

No problem on Umbrel OS* :D

<img width="1377" alt="Image" src="https://github.com/user-attachments/assets/c14c2561-7a25-44da-981e-20c48bf194a6" />

**Comment by ErikBjare** [comment]
@hcastc00 maybe?
